### PR TITLE
isDisabled prop

### PR DESCRIPTION
**Pull request by jpranays**
This PR contains an optional prop **isDisabled**, which disables all the inputs and tabs, making them non-interactive.

### CODE REVIEW

File: src/lib/cron-tab/custom.tsx [hunk 1]
```diff
@@ -4,18 +4,20 @@ interface CustomCronProp {
     onChange(e: string[]): void
     value: string[]
     translate(e: string): string
+    isDisabled?: boolean
 }
 
 const CustomCron: React.FunctionComponent<CustomCronProp> = (props) => {
     const onChange = (e: {target: { value: string}}) => {
+        if(props.isDisabled===true) return;
```
Comment (sojinantony01) [inline src/lib/cron-tab/custom.tsx:12]: Do we really need === true check?
if(props.idDisabled) is enough rt ?

File: src/lib/cron-tab/custom.tsx [hunk 2]
```diff
@@ -4,18 +4,20 @@ interface CustomCronProp {
     onChange(e: string[]): void
     value: string[]
     translate(e: string): string
+    isDisabled?: boolean
```
Comment (sojinantony01) [inline src/lib/cron-tab/custom.tsx:7]: can you change it to generic key "disabled" ?

File: src/lib/cron-tab/daily.tsx [hunk 3]
```diff
@@ -20,6 +21,7 @@ const DailyCron: FunctionComponent<DailyCronProp> = (props) => {
     }, [])
 
     const onDayChange = (e: {target: { value: string}}) => {
+        if(props.isDisabled === true) { return }
```
Comment (sojinantony01) [inline src/lib/cron-tab/daily.tsx:24]: disabled prop to input box will stop triggering onchange rt? 
Do we need this check?

**Review by sojinantony01 — CHANGES_REQUESTED** [review]

File: src/lib/cron-tab/custom.tsx (hunk 1 above)
Comment (jpranays) [reply src/lib/cron-tab/custom.tsx]: Yes, it's enough, no extra type-check is reqd.

File: src/lib/cron-tab/custom.tsx (hunk 2 above)
Comment (jpranays) [reply src/lib/cron-tab/custom.tsx]: Okay

File: src/lib/cron-tab/daily.tsx (hunk 3 above)
Comment (jpranays) [reply src/lib/cron-tab/daily.tsx]: No, just **disabled** attribute to input box is not enough, malicious user could remove the **disabled** attr from dev tools, thus resulting in state change,
That's why extra JS check was added.

**Comment by jpranays** [comment]
I have made requested changes, please do review once.

**Comment by jpranays** [comment]
Kindly do review new changes

**Review by sojinantony01 — APPROVED** [review]

**Comment by sojinantony01** [comment]
Thanks for the PR. 
I have updated the documentation part as well(ReadMe)

**Pull request merged**